libsolidity/interface/Natspec.cpp

```diff
@@ -37,6 +37,9 @@ Json::Value Natspec::userDocumentation(ContractDefinition const& _contractDef)
 	Json::Value doc;
 	Json::Value methods(Json::objectValue);
 
+	doc["version"] = Json::Value(NATSPEC_VERSION);
+	doc["kind"]    = Json::Value("user");
```

Comment:
Is this our style? I would guess we don't use the extra spaces

Reply:
Ah, no it isn't. Old habit. Will fix.